python/doc/source/development.rst

```diff
@@ -125,9 +125,13 @@ dependencies will be automatically built by Arrow's third-party toolchain.
                           libboost-filesystem-dev \
                           libboost-system-dev \
                           libboost-regex-dev \
+                          python-dev \
+                          autoconf \
                           flex \
                           bison
 
+If you are building Arrow for Python 3, install ``python3-dev`` instead of ``python-dev``.
```

Comment:
If both are installed, does the user need to pass some extra things to cmake to select the one they want (e.g. `-DPYTHON_EXECUTABLE`)?

Reply:
Exactly! I added an explanation about that to the PR. Thanks for the comment.

Reply:
Oh, I accidentally closed this. I'll submit it as another PR.